ghorgs/wrappers.py

```diff
@@ -143,7 +143,7 @@ def get_paged_content(self, session: requests.Session, url: str) -> list:
             assert isinstance(data, list)
             all_data.extend(data)
 
-            # Archie> Need to add this check here so last page will be processed
+            # Archie> Need to move this check here so last page will be processed for project
```

Comment:
Can you remove your name here, and actually the command.
These types of comments are best for the commit mesasge.

Reply:
Oops sorry that was my personal comment during debugging :P